entity/src/sbom_package_license.rs

```diff
@@ -0,0 +1,57 @@
+use sea_orm::entity::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel)]
+#[sea_orm(table_name = "sbom_package_license")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: Uuid,
+    pub sbom_id: Uuid,
+    pub node_id: String,
+    pub license_id: Uuid,
+    pub license_type: LicenseCategory,
+}
+
+#[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
+pub enum Relation {
+    #[sea_orm(
+        belongs_to = "super::sbom::Entity",
+        from = "Column::SbomId",
+        to = "super::sbom::Column::SbomId"
+    )]
+    Sbom,
+    #[sea_orm(has_many = "super::sbom_package::Entity")]
+    Package,
+    #[sea_orm(has_one = "super::license::Entity")]
+    License,
+}
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, EnumIter, DeriveActiveEnum)]
+#[sea_orm(rs_type = "String", db_type = "Enum", enum_name = "license_category")]
+pub enum LicenseCategory {
+    #[sea_orm(string_value = "slc")]
+    SpdxDeclared,
+    #[sea_orm(string_value = "sld")]
+    SpdxConcluded,
```

Comment:
Are these annotations correct?  sld would make more sense to mean declared to me.

Reply:
@bxf12315 beside being switched as Hiram reported, @ctron also clarified having `Spdx` is useless (ref. https://github.com/trustification/trustify/pull/1331/files#r1967240377) so please rename them in `Declared` and `Concluded` so that they'll be used for both SPDX and CycloneDX SBOMs.